package.json

```diff
@@ -40,16 +40,19 @@
     "build": "tsc",
     "build:browser": "npm run build && webpack --config webpack.config.cjs && webpack --config webpack.config.esm.cjs",
     "watch": "tsc -w",
-    "lint": "ts-standard src/ utils/",
-    "lint:fix": "ts-standard src/ utils/ --fix",
-    "test": "npm run lint && npm run build && npm run test:node && npm run test:browser",
+    "lint": "tsc && biome check --no-errors-on-unmatched --files-ignore-unknown=true .",
+    "lint:fix": "biome check --no-errors-on-unmatched --files-ignore-unknown=true --fix .",
     "test:ci": "npm run test",
-    "test:node": "mocha 'dist/test/**/*.js'",
-    "test:browser": "polendina --page --cleanup dist/test/**/*.test.js",
-    "test:watch": "npm run build && mocha 'dist/test/**/*.js' --watch",
+    "test": "npm run lint && npm run test:node && npm run test:browser",
+    "test:node": "playwright-test 'src/test/**/!(*.browser).test.{js,ts}' --mode node",
+    "test:browser": "playwright-test 'src/test/**/!(*.node).test.{js,ts}'",
+    "test:watch": "npm test:node --watch",
     "clean": "rm -rf dist",
     "prepublishOnly": "npm run clean && npm run build && npm run build:browser"
   },
+  "simple-git-hooks": {
```

Comment:
I already hate this a little bit; it didn't work for me because I use /usr/local/bin and that's not included in the basic hook env. https://github.com/toplenboren/simple-git-hooks?tab=readme-ov-file#i-am-getting-npx-command-not-found-error-in-a-gui-git-client is kind of a lame solution to this but I've just manually modified it. The problem I have with it is that it's hostile to contributors who will be surprised to find a hook magically appear and then not know how to fix it.

Reply:
but its optional, it doesnt install automatically

Reply:
https://github.com/FilOzone/synapse-sdk/pull/177

Reply:
I'm getting used to it being there now that I have it fixed and working, but this is still hostile for new contributors as it is

Reply:
wow my guess, is that i always use pnpm and it does not run any of those scripts by default.

should we remove it ?

Reply:
I'm not seeing why it's needed as a devdep, it doesn't get pulled in to run the hook, just to set it up and we have the docs for doing that in the README so removing it makes sense to me